UnityProject/Assets/Scripts/Doors/WindoorAnimator.cs

```diff
@@ -0,0 +1,246 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+using UnityEditor;
+
+/// <summary>
+/// Used for Windoors
+/// Windoor animator.
+/// Remember to name at least one of the child sprite renderers as 'DoorBase'
+/// </summary>
+public class WindoorAnimator : DoorAnimator
+{
+	[Tooltip("The resource path to the sprite sheet. i.e: icons/obj/doors/windoor")]
+	public string WspritePath;
+
+	[Tooltip("A list of frame numbers for the open/close animation, not including the openFrame and closeFrame")]
+	public int[] WanimFrames;
+
+	public int WanimLength;
+	public int WcloseFrame;
+	public int WdeniedFrame;
+	public int WopenFrame;
+
+	public bool IncludeAccessDeniedAnim;
+	[Tooltip("Only check if the door changes appearance (smooths) based on nearby walls")]
+	public bool Smoothed;
+	private Tilemap tilemap;
```

Comment:
Codacy found an issue: [Remove the field 'tilemap' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)